test/stripStyle-test.js

```diff
@@ -0,0 +1,15 @@
+import stripStyle from '../src/stripStyle';
```

Comment:
Should I just include the test for ignoring `undefined` in this PR and put the other 2 tests in a new PR?